src/nni_manager/training_service/pai/paiJobRestServer.ts

```diff
@@ -21,17 +21,17 @@ export interface ParameterFileMeta {
  */
 @component.Singleton
 export class PAIJobRestServer extends ClusterJobRestServer {
-    private parameterFileMetaList: ParameterFileMeta[] = [];
+    protected parameterFileMetaList: ParameterFileMeta[] = [];
 
     @Inject
-    private readonly paiTrainingService: PAITrainingService;
+    protected readonly paiTrainingService: PAITrainingService;
```

Comment:
`@Inject` should be removed if it is  assigned in constructor.

Reply:
fixed.